Types: cannot find module 'big.js' or its corresponding type declarations

In the [type declaration file](https://github.com/Dintero/money/blob/master/dist/index.d.ts#L1), the types from `big.js` are imported. But `@types/big.js` is [only included in `devDependencies`](https://github.com/Dintero/money/blob/master/package.json#L43), so those who install this package will not automatically install `@types/big.js` too. If they use TypeScript, the type-checker will throw errors when checking the type declaration file, because `@types/big.js` is missing.

```
error TS2307: Cannot find module 'big.js' or its corresponding type declarations.
```

If types packages are used in type declaration files that are published to NPM, then those type packages should be included in `dependencies` instead of `devDependencies`.

--- a/package.json
+++ b/package.json
@@ -38,13 +38,13 @@
         ]
     },
     "dependencies": {
+        "@types/big.js": "6.2.2",
         "big.js": "^6.1.1",
         "currency-codes": "^2.1.0"
     },
     "devDependencies": {
         "@eslint/js": "9.7.0",
         "@semantic-release/git": "10.0.1",
-        "@types/big.js": "6.2.2",
         "@types/jest": "29.5.12",
         "eslint": "9.7.0",
         "eslint-config-prettier": "9.1.0",

